storage/postgres/admin_storage.go

```diff
@@ -52,6 +51,13 @@ const (
 		deleted,
 		delete_time_millis
 	FROM trees`
+
+	nonDeletedWhere       = " WHERE (deleted is NULL OR deleted = false)"
```

Comment:
Is there a reason for `deleted` being nullable?

Reply:
The PG schema followed the example of `storage.sql` for MySQL: https://github.com/google/trillian/blob/master/storage/mysql/storage.sql#L23-L24 if you think that this shouldn't be the case let me know.

Reply:
I'm not sure why it was made nullable for MySQL. It may have been because it was added to the schema later. In this case, I don't see any reason to make it nullable and it is causing more code complexity.